tools/depends/native/Makefile

```diff
@@ -11,7 +11,7 @@ NATIVE= m4 gettext heimdal autoconf automake \
         libpng libjpeg-turbo liblzo2 giflib \
         setuptools distutilscross JsonSchemaBuilder TexturePacker \
         flatbuffers \
-        meson ninja
+        meson ninja MarkupSafe Mako
```

Comment:
Is there a need to add these to every native platform?
Would the GBM target ever be compilable on Darwin (OSX)?

Reply:
yea good catch thanks!